external/kinesis-asl/src/main/scala/org/apache/spark/streaming/kinesis/KinesisInputDStream.scala

```diff
@@ -100,6 +103,7 @@ object KinesisInputDStream {
     private var endpointUrl: Option[String] = None
     private var regionName: Option[String] = None
     private var initialPositionInStream: Option[InitialPositionInStream] = None
+    private var initialPositionInStreamTimestamp: Option[Date] = None
```

Comment:
@brkyvz Where exactly are we planning to add these changes. Are you proposing to change the type of 
`private var initialPositionInStreamTimestamp: Option[Date] = None`

That would introduce a backward incompatibility on the current builder ?